packages/morph/src/styles/editor.scss

```diff
@@ -253,9 +259,12 @@ html {
         transition: transform 0.2s ease, font-weight 0.2s ease;
         font-family: $font-body;
         font-weight: $normalWeight;
+        html.dark & {
+          background-color: #010042; // Dark mode
+          color: #e0e0e0; // Dark mode text
+        }
```

Comment:
ugh thi kinda defeats the whole purpose.

a better solution is to control it through `var(--<colorscheme>)`